programs/github-pull-request/getGithubAppUserToken/index.js

```diff
@@ -0,0 +1,18 @@
+const GitHubApp = require('../../../lib/GithubApp');
+
+/**
+ * Get a github app installation token.
+ * @param  {String} options.appId
+ * @param  {String} options.appPrivateKey
+ * @return {Promise<String | undefined>}
+ */
+module.exports = async function getGithubAppUserToken({appId, appPrivateKey}) {
+	if (!(appId && appPrivateKey)) {
```

Comment:
Why we empty return here, Looking on this file, on a isolated manner, it looks a bit weird.

Regardless I belive, if instansinating `GitHubApp` without the proper appId / appSecret, It should throw. ( I know this is done on toper level)

Reply:
Yea but I can throw here as well

Reply:
I rather the constrctor throws, as it part of the Instansination flow